tests/test_ancillaries.py

```diff
@@ -72,6 +75,115 @@ def test_af_deform():
     assert np.allclose(ds["deform"], 1 - ds["circ"])
 
 
+def test_af_get_bright_called_only_once(monkeypatch):
+    """Test that `get_bright` is only called once, which is desired when
+    creating ancillary features that share the same pipeline"""
+    def wrapper_on_get_bright(mm):
```

Comment:
Can't you somehow decorate this wrapper with `Mock` somehow and then monkeypatch `get_bright` with it?
```python
old_get_bright = dclab.features.bright.get_bright
@some_mockdecorator
def wrapper_get_bright(mm):
   return old_get_bright(mm)
monkeypatch.setattr('dclab.features.bright.get_bright', wrapper_get_bright)
ds["bright_sd"]
ds["bright_avg"]
wrapper_get_bright.assert_called_once()
wrapper_get_bright.reset_mock()
```
Instead of `some_mock_decorator` you could probably also use the original `calltracker`.